patches/chrome-browser-resources-settings-basic_page-basic_page.html.patch

```diff
@@ -1,18 +1,21 @@
 diff --git a/chrome/browser/resources/settings/basic_page/basic_page.html b/chrome/browser/resources/settings/basic_page/basic_page.html
-index 65d9633a7c0e4b0b44dd8a0ae2bb7598b3ee1b06..c6a6980abc74d80c8d3b8fa9104c6494f113f1e4 100644
+index 65d9633a7c0e4b0b44dd8a0ae2bb7598b3ee1b06..22e9e8f6445789c8c9ff90dc20bce01eabd8cb96 100644
 --- a/chrome/browser/resources/settings/basic_page/basic_page.html
 +++ b/chrome/browser/resources/settings/basic_page/basic_page.html
-@@ -27,6 +27,9 @@
+@@ -27,7 +27,11 @@
  
  <if expr="not chromeos">
  <link rel="import" href="../default_browser_page/default_browser_page.html">
+-</if>
 +<link rel="import" href="../default_brave_shields_page/default_brave_shields_page.html">
```

Comment:
we should import a single file that contains other imports cc @petemill

Reply:
Moved to `privacy_page`. In there, we have a `brave_privacy` page for additions and `brave__rewards_privacy` lives in there.